site/profile/manifests/proxy_puppet.pp

```diff
@@ -94,7 +94,7 @@
   haproxy::balancermember { 'foreman-backend8888':
     listening_service => 'foreman-frontend8888',
     server_names      => 'foreman.localdomain',
-    ipaddresses       => '172.28.128.22',
+    ipaddresses       => '172.28.128.20',
```

Comment:
I didn't notice any IP changes in the Vagrantfile of the other repo so how come this is changing?